### ISSUE TITLE
Ability to add waypoints in between start and end

### ISSUE THREAD

**Issue by amot-dev** [amot-dev/mototwist#102]
Most of the time, shaping points are an afterthought and only added after realizing the calculated route is not the desired route. A few ways of solving this:
- Preset waypoint placing order of Start -> End -> Shaping 1 -> Shaping 2 -> etc..
  - Potentially very confusing for new users, may need a persistent mouse tooltip to notify what the next placement will be
- Reorganize waypoints button
  - High Friction
- Button in waypoint menu to "Add After"
  - Difficult to find, again confusing on what is being placed next (does this "Add After" persist when deleting newly added waypoint? does it matter since waypoints can/should be moved instead of re-created?)
